### PR TITLE
Image support

### PR DESCRIPTION
**Pull request by victorcuena**

### CODE REVIEW

File: src/blocks/image.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    const mjmlProperties = item.properties
+    const properties = propertiesToText(mjmlProperties);
+
+    return `<mj-image ${properties} />`;
+};
+
+const imageToText = (item) => {
+    const alt = item.properties.alt || 'image';
+    const href = item.properties.href ? `(${item.properties.href})` : '';
+    return `[${alt}]${href}`;
```
Comment (istanishev) [inline src/blocks/image.js]: We have the link if [] for the others.

File: src/blocks/image.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { propertiesToText } from './common/base';
+
+const imageToMjml = (item) => {
+    const mjmlProperties = item.properties
+    const properties = propertiesToText(mjmlProperties);
```
Comment (istanishev) [inline src/blocks/image.js]: Any properties to "translate" or "skip"?

Comment (istanishev) [reply src/blocks/image.js]: For example: 'container-background-color'?

Comment (istanishev) [reply src/blocks/image.js]: It would be nice to support border as well (similar to the other components)

File: src/blocks/image.js (hunk 1 above)
Comment (victorcuena) [reply src/blocks/image.js]: I commit some changes to follow the markdown syntax.

Comment (victorcuena) [reply src/blocks/image.js]: I'm not sure what do you mean with adding new lines.

File: src/blocks/image.js (hunk 2 above)
Comment (victorcuena) [reply src/blocks/image.js]: The only properties that I'm using are `src`, `width`, `alt`, `href` which don't need any translation.

Comment (victorcuena) [reply src/blocks/image.js]: Added support for borders.

**Pull request merged**